python/graphstorm/model/gnn.py

```diff
@@ -711,7 +711,7 @@ def restore_sparse_model(self, restore_model_path):
         # restore sparse embeddings for node_input_encoder.
         load_sparse_embeds(restore_model_path, self.node_input_encoder)
 
-    def init_optimizer(self, lr, sparse_optimizer_lr, weight_decay, lm_lr=None):
+    def init_optimizer(self, lr, sparse_optimizer_lr=0.01, weight_decay=0, lm_lr=None):
```

Comment:
You can set default value of sparse_optimizer_lr to None and check if it is None, use lr.

Reply:
Agree. Changed.